content/blog/nats-server-2.9.17-release.md

```diff
@@ -0,0 +1,124 @@
+
+However, the upside of addressing this issue resulted in more (correct) optimizations for the WebSocket transport, including reducing the overall memory footprint and switching to a [faster compression algorithm](https://github.com/nats-io/nats-server/pull/4087).
+
+**Relevant PRs**
+
+- [#4084](https://github.com/nats-io/nats-server/pull/4084)
+- [#4087](https://github.com/nats-io/nats-server/pull/4087)
+- [#4093](https://github.com/nats-io/nats-server/pull/4093)
+
+## ⚕️ Health monitor improvements
+
+NATS provides a set of [monitoring endpoints](https://docs.nats.io/running-a-nats-service/nats_admin/monitoring) containing a variety of information about the current state of the server.
+
+The job of the `/healthz` endpoint is to return `200 OK` when the server is ready for client connections and, if JetStream is enabled, all of the assets (streams and consumers) on the server have been accounted for, any necessary recovery was performed, and the integrity of the data has been verified.
+
+In rare situations, for clusters with a large number of assets, or if the asset sizes are large, it was possible for given server to get stuck and never finish the internal checks, therefore never becoming ready for client connections. A forced server restart would typically fix this issue, however, clearly this was not ideal.
+
+This release brings an internal routine that periodically checks whether all assets have checked and restarts the ones that have become stuck. This obviates the need to force restart the server, resulting in a full re-check and further improves NATS ability to self-heal.
+
+**Relevant PRs**
+
+- [#4116](https://github.com/nats-io/nats-server/pull/4116)
+- [#4172](https://github.com/nats-io/nats-server/pull/4172)
+
+## 🍃 Leafnode fleet optimizations
+
+For edge-related use cases, it is becoming increasingly common for each edge target to have a _leafnode cluster_ deployed rather than a single [leafnode](https://docs.nats.io/running-a-nats-service/configuration/leafnodes). This provides high availability and fault tolerance of stream data (including Key-Value and object store) for locally connected services.
+
+This is possible by configuring the leafnodes as a local [cluster](https://docs.nats.io/running-a-nats-service/configuration/clustering) while each node also having the [`remotes`](https://docs.nats.io/running-a-nats-service/configuration/leafnodes/leafnode_conf#leafnode-remotes-entry-block) block configured establishing one or more connections to the remote cluster.
+
+As the scale of edge deployments increase, so does the need to optimize how real-time propagation of _subject-interest_. Where it was common to have many single leafnode deployments, having clusters of three or greater for edge deployment 3x-5x's the number of nodes in the fleet. For use cases such as electric vehicles and charging stations, the number of edge locations and leafnodes can be quite high!
```

Comment:
Because it's "the scale of X", it's singular, so "increase" -> "increases". Then, end of this sentence, you're missing the verb. Insert "happens".